src/aides.yaml

```
@@ -612,27 +612,52 @@ aides . mons en baroeul:
   titre: Ville de Mons en Baroœul
```

j'avais fais une coquile

```suggestion
  titre: Ville de Mons en Barœul
```